extensions/sql-database-projects/package.json

```diff
@@ -388,6 +397,11 @@
           "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.file.externalStreamingJob",
           "group": "5_dbProjects_streamingJob"
         },
+        {
+          "command": "sqlDatabaseProjects.openInDesigner",
+          "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.file.table && config.workbench.enablePreviewFeatures",
```

Comment:
should also have the condition `azdataAvailable` like other features only available in ADS, like schema compare and update project from database commands in this file